src/index.ts

```diff
@@ -73,6 +72,15 @@ integration.addBuildEventHandler(
       }
     }
 
+    console.log("Config:");
+    console.log("---");
+    console.log(`Report Only: ${config.reportOnly}`);
+    console.log(`Report URI: ${config.reportUri}`);
+    console.log(`Unsafe Eval: ${config.unsafeEval}`);
+    console.log(`Path: ${config.path.join(", ")}`);
+    console.log(`Excluded Path: ${config.excludedPath.join(", ")}`);
```

Comment:
are we sure that `path` and `excludedPath` will always be arrays here? they can be strings if defined in the UI, no?

Reply:
they get parsed into arrays on save!